### PR TITLE
chore: org update

### PR DESCRIPTION
**Pull request by frezbo**
Org update

Signed-off-by: Noel Georgi <git@frezbo.dev>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -96,4 +96,4 @@ kernel-%: ## Updates the kernel configs: e.g. make kernel-olddefconfig; make ker
 
 .PHONY: conformance
 conformance: ## Performs policy checks against the commit and source code.
-	docker run --rm -it -v $(PWD):/src -w /src ghcr.io/talos-systems/conform:v0.1.0-alpha.22 enforce
+	docker run --rm -it -v $(PWD):/src -w /src ghcr.io/siderolabs/conform:latest enforce
```
Comment (rgl) [inline Makefile:99]: was changing to the `latest` version really intended?

Comment (frezbo) [reply Makefile:99]: `conform` is for CI tests, shouldn't hurt :sweat_smile:

Comment (frezbo) [reply Makefile:99]: on second thoughts since this is a backport makes sense to only change the org name, thanks for the pointer :+1:

**Review by smira — APPROVED** [review]

**Pull request merged**